Projects/GLU/GLU01/GLU01-AWS100.md

```diff
@@ -0,0 +1,43 @@
+# GLU02-AWS100 - Introduction to SNS (Simple Notification Service)
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+- [Edward Allen Mercado](https://twitter.com/edwardmercado_)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create an SNS topic
+- Subscribe to that SNS topic with an email adress of yours
+- Send a test message through the SNS topic
+- Make sure you received the message to your email adress
```

Comment:
My bad, "address" is double "d"😅 can you change these please? That's on me haha.

Reply:
no worries, it has been modified.